docs/source/requests.md

```diff
@@ -132,7 +132,7 @@ Each Request (both write and read) is a JSON with a number of common metadata fi
  
 - `reqId` (integer): 
 
-    Unique ID number of the request with transaction.
+    Unique ID number of the requested transaction.
```

Comment:
Same problem as in https://github.com/hyperledger/indy-node/pull/1497

Reply:
This was the wording we settled on for #1497, does the same work here?
```suggestion
    Unique ID number of the request containing a transaction to be written.
```